package/package.json

```diff
@@ -78,7 +78,7 @@
     "path": "0.12.7",
     "react-native-markdown-package": "1.8.2",
     "react-native-url-polyfill": "^1.3.0",
-    "stream-chat": "8.17.0"
+    "stream-chat": "8.21.0"
```

Comment:
Any reason why you aren't using the latest LLC?

Reply:
No reasons. We haven't tested it yet, and that is why didn't upgrade to the latest.

Reply:
I think when updating a dependency, we should always update to the latest version. We test it anyways, this way we can cover a wider version range.

Reply:
we usually update to the latest.. @khushal87 could you upgrade and quickly check.. nothing should break with a patch release update

Reply:
Updated to the latest version. I don't see any problem as such as of now.